### PR TITLE
Minor translation - 17251 - enhance gem error message, and 871- minor…

### PR DESCRIPTION
**Pull request by merrun**
… name fixing

17251 강화 할 수 없는 젬입니다. - This gem cannot be enhanced.

871 mentiwood > mantiwood
since the all the drop regarding mentiwood refers it to mantiwood instead of mentiwood

please don't conflict again 
if still conflict again later on ..I gave up ..

### CODE REVIEW

**Comment by Sunaries** [comment]
:+1: good to merge

File: ETC.tsv [hunk 1]
```diff
@@ -17246,7 +17246,7 @@ ETC_20151102_017247	You have been restricted from chatting until {Time}.
 ETC_20151102_017248	You have been restricted from trading until {Time}.
 ETC_20151102_017249	The .ipf files are damaged and cannot be read. The client will now shutdown.
 ETC_20151102_017250	You cannot use the gem on this equipment.
-ETC_20151102_017251
+ETC_20151102_017251 This gem cannot be enhanced.
```
Comment (grillo-delmal) [inline ETC.tsv]: Careful with the spacing here, there should be a tab in between, not a space

**Comment by merrun** [comment]
I did use tab ...but it become that

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**